include/clap/ext/draft/midi-info.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension lets a plugin expose it's MIDI implementation.
+// The primary goal is to allow a host to create a MIDI-CI Property Exchange "ChCtrlList" resource for the plugin,
+// so keyboard controllers can assign their knobs to suitable controllers.
+
+//TODO : clap_midiinfo could contain more (optional) info (min/max, stepcount, paramPath etc.)

```

Comment:
This information is all in the param; and this extension won't work if you don't have param. So can you just glean it from there?

Reply:
This extension isn't related to params at all I think. Why do you think so?

Reply:
Specifically what's the relationship between this extension and

https://github.com/free-audio/clap/blob/main/include/clap/ext/draft/midi-mappings.h

the midi-mappings extension proposed here?

Reply:
midi-mappings.h doesn't provide a way to say how important a param is. So a plugin would have to list any mapped params in midi-info.h again.

One could argue these two extensions could/should be rolled into one.

Reply:
Well, the two extensions aren't really about the same thing: midi-mappings.h is about describing which param/MIDI controller pairs do the same thing. midi-info.h is about describing which MIDI messages work and what the important ones are.